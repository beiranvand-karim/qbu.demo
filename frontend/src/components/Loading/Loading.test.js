/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {Loading} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"

describe('<Loading />', () => {
   it('should render without crashing', () => {
      shallow(<Loading/>)
   });
   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><Loading/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});