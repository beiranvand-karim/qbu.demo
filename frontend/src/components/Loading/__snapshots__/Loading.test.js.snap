// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should matches the snapshot 1`] = `
<h1>
  loading ...
</h1>
`;
